SU-HSDO/suhumsci
# Fix(STN-952):  Fix credit cap overlay horizontal raised cards

# READY FOR REVIEW

## Summary
A bug was found from hotfix #997, it was causing the caption/credit overlay to go full height due to the wildcard we had to put back in for prior issues.

![image (2)](https://user-images.githubusercontent.com/9355698/151018648-611cdf36-8903-4f7d-9f77-cd09da00d1a8.png)

JIRA Card: https://sparkbox.atlassian.net/browse/STN-952

This work adds a `not()` to the wildcard so it does not apply to the image caption overlay div. This was the smallest way I could see of nudging that work without getting too involved with full removal as that caused issues prior.

## Need Review By (Date)
asap

## Urgency
medium

## Steps to Test
1. npm run test
2. Go the the "News" View page with horizontal cards.
3. Add `hb-raised-cards' to the Layout of that main region block for that page.
4. Make sure caption/credits are added to those images in the News article, if they aren't already.
5. Confirm the overlay is using `height:auto` as intended and not being overridden by the wildcard class to `height:100%`.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
